lib/wwwtxt/plain_text_renderer.rb

```diff
@@ -0,0 +1,123 @@
+require "htmlentities"
+
+module WwwTxt
+  class PlainTextRenderer
+    FORMAT = "txt"
+
+    def self.render_from_html(html)
+      strip_html(html)
+    end
```

Comment:
I'll probably end up adding more steps than just stripping the html. Nevertheless it should be purely functional, so perhaps module functions are the way to go.

The other consideration is that I might have other classes, like a `PDFRenderer#render_from_html`, but I suppose I can pass in a module just like I'm passing in a class.